### PR TITLE
Device added: Logitech MX Master 3S

### PR DESCRIPTION
**Pull request by NL-TCH**
Tested + working

See my pullrequest for the libratbag support: https://github.com/libratbag/libratbag/pull/1346

### CODE REVIEW

**Comment by NL-TCH** [comment]
image

**Comment by omarjpeg** [comment]
Does this version support the thumb button and moving the mouse in different directions to do different actions?
Thank you

**Comment by NL-TCH** [comment]
Hey @omarjpeg 

No i am sorry but the current driver of libratbag does not support that functionality.
For the configuration of the thumbwheel + palmrestbuttom i would you advise to look at Solaar
Solaar github: https://github.com/pwr-Solaar/Solaar
Solaar homepage: https://pwr-solaar.github.io/Solaar/

Kind regards

Teunis

**Comment by omarjpeg** [comment]
@NL-TCH 
No worries at all, thank you so much for the contribution and suggestion :)

File: data/svgs/svg-lookup.ini [hunk 1]
```diff
@@ -170,6 +170,10 @@ Svg=logitech-mx-ergo.svg
 DeviceMatch=bluetooth:046d:b023;usb:046d:4082
 Svg=logitech-mx-master-3.svg
 
+[Logitech MX Master 3S]
+DeviceMatch=bluetooth:046d:b034;usb:046d:c548
```
Comment (staticssleever668) [inline data/svgs/svg-lookup.ini:174]:
```suggestion
DeviceMatch=bluetooth:046d:b034
```
Please, remove the receiver ID `c548` (following https://github.com/libratbag/libratbag/pull/1346#discussion_r985252293).

**Comment by kodein1** [comment]
Piper and libratbak were updated today, but Mx3s is still not there. How does it look with adding this mouse to piper?.

**Comment by staticssleever668** [comment]
@kodein1, hi, before merging this I want to enable support on libratbag side. There was a PR on libratbag side which I closed by accident and can't open it again until it's creator reopens it themselves. (https://github.com/libratbag/libratbag/pull/1346)

Adding this mouse doesn't require any changes to the driver, it's just the device file that's added, so, until these PRs get merged and we release new versions of libratbag and Piper, you can enable it's support by simply creating this device file on your system.

Create a file named `logitech-MX-Master-3S.device` and put it under `/usr/share/libratbag`.
```dosini
[Device]
Name=Logitech MX Master 3S
DeviceMatch=bluetooth:046d:b034
Driver=hidpp20
```

**Review by staticssleever668 — APPROVED** [review]

**Comment by kodein1** [comment]
Does this mean mx3s will finally be added?

**Comment by staticssleever668** [comment]
Thank you!

**Pull request merged**